### ISSUE TITLE
[Android] SVG as Icon

### ISSUE THREAD

**Issue by hallahan** [mousebird/WhirlyGlobe#717]
I attempted to use an SVG imported into an android project as an `R.drawable`. PNGs work fine, but SVG does not seem to work.

I was experimenting with using OpenStreetMap icons.

https://github.com/gravitystorm/openstreetmap-carto/tree/master/symbols

In particular:

https://github.com/gravitystorm/openstreetmap-carto/blob/master/symbols/town_hall.16.svg

**Comment by mousebird** [comment]
Yup, we don't do anything with SVGs.  Can only handle images for icons.
